### DISCUSSION TITLE
Search Web Parts stopped appearing on page

### DISCUSSION THREAD

**Discussion by jack-vinitsky** [microsoft-search/pnp-modern-search#2506] [Q&A] [answered]
Greetings,

We recently updated to 4.7 from the prior version.  Sometime after the PnP search web parts stopped appearing on pages they were previously placed on AND also within the "Add a Web Part" toolbox.

I can see the outlines of the web parts when in Edit Mode but nothing appears in the property pane. When I click on the Edit pencil I get the below:

![image](https://user-images.githubusercontent.com/74683257/199545368-70dba234-e687-4877-8aeb-af656f7a26d4.png)

Web parts are NOT deployed globally.

This seem to be happening on 2 specific sites (separate site collections but in the same Hub). However when I open a different browser tab on another site that has the PnP web part on it but does not seem to have this issue, and then go and refresh the page on one of the problem sites, **the web parts reappear (both on page and in the add a web part tool pane)**.  However if I close the browser entirely and then go back to the problem site the issue reappears.

I have tried the following:

1. Reinstalling 4.7
2. Removing and re-adding the web parts from the page.
3. Removing the app from the problem site and adding it back.
4. Clearing browser cache

Would love some help on this if anyone has experienced something like this before.

Regards,

Jack Vinitsky

**Comment by wobba** [comment, marked as answer]
See this tweet chain for info: https://twitter.com/NelloDAndrea/status/1587527135702441984 

What's the previous version you used? Resolution is to delete sppkg, re-add the new version. Remove solution from site, and re-add it. All webparts should continue to work as is.

**Comment by jack-vinitsky** [comment]
Thanks so much for the quick response.

According to my notes, the prior version that we had deployed was 4.4.1.

Will try your suggestion.

**Comment by jack-vinitsky** [comment]
@wobba Your suggested steps have resolved the issue.  Thanks again!
